src/api.c

```diff
@@ -488,6 +496,65 @@ int trixi_nelements_global(int handle) {
 }
 
 
+/**
+ * @anchor trixi_ndofs_api_c
+ *
+ * @brief Return number of local degrees of freedom.
+ *
+ * These usually differ from the global count when doing parallel computations.
+ *
+ * @param[in]  handle  simulation handle
+ *
+ * @see trixi_ndofs_global_api_c
+ */
+int trixi_ndofs(int handle) {
+
+    // Get function pointer
+    int (*ndofs)(int) = trixi_function_pointers[TRIXI_FPTR_NDOFS];
+
+    // Call function
+    return ndofs(handle);
+}
+
+
+/**
+ * @anchor trixi_ndofs_global_api_c
+ *
+ * @brief Return number of global degrees of freedom.
+ *
+ * These usually differ from the local count when doing parallel computations.
+ *
+ * @param[in]  handle  simulation handle
+ *
+ * @see trixi_ndofs_api_c
+ */
+int trixi_ndofs_global(int handle) {
```

Comment:
Most of my comments re naming conventions etc. that I left above also apply to the C/Fortran API, so I won't repeat them here